assets/js/pages/Layout/Layout.jsx

```diff
@@ -84,9 +84,8 @@ function Layout() {
       : localStorage.setItem('sidebar-collapsed', true);
   }, [isCollapsed]);
 
-  const email = useSelector(selectUserEmail);
-  const username = useSelector(selectUserName);
-
+  const userProfile = useSelector(getUserProfile);
```

Comment:
We could destruct user Profile for username and email and pass it as props for profile menu
``` const { username, email } = useSelector(getUserProfile);```